README.md

```diff
@@ -17,48 +14,44 @@ Jump in: [![Slack Status](http://slack.samvera.org/badge.svg)](http://slack.samv
 - [Ingesting Content](#ingesting-content)
 - [Developing, Testing, and Contributing](#developing-testing-and-contributing)
   - [Contributing](#contributing)
 - [Acknowledgements](#acknowledgements)
-  - [Sponsoring Organizations](#sponsoring-organizations)
-  - [More Information](#more-information)
-  - [Contact](#contact)
 
 <!-- /TOC -->
 
 # Overview
-IiifPrint is a gem (Rails "engine") for [Hyrax](https://hyrax.samvera.org/) -based digital repository applications to support ingest, management, and display of digitized newspaper content.
+IiifPrint is a gem (Rails "engine") for [Hyrax](https://hyrax.samvera.org/)-based digital repository applications to support displaying parent/child works in the same viewer (Universal Viewer) and the ability to search OCR from the parent work to the child work(s).
 
-IiifPrint is not a stand-alone application. It is designed to be integrated into a new or existing Hyrax (v2.5-v2.9.4) application, providing content models, ingest workflows, and feature-rich UX for newspaper repository use-cases.
+IiifPrint is not a stand-alone application. It is designed to be integrated into a new or existing Hyku (v4.0-v5.0) application.  Future development will include integrating it into a Hyrax-based application without Hyku and support for [IIIF Presentation Manifest version 3](https://iiif.io/api/presentation/3.0/) along with [AllinsonFlex](https://github.com/samvera-labs/allinson_flex) metadata profiles.
 
 IiifPrint supports:
-* models for Title, Issue, Page, and Article
-* batch ingest via command line
 * OCR and ALTO creation
-* newspaper-specific metadata fields
 * full-text search
-* calendar-based issue browsing
-* advanced search
 * OCR keyword match highlighting
 * viewer with page navigation and deep zooming
+* splitting of PDFs to LZW compressed TIFFs for viewing
+* configuring how the manifest canvases are sorted in the viewer
+* adding metadata fields to the manifest with faceted search links and external links
+* excluding specified work types to be found in the catalog search
 
-A complete list of features can be found [here](https://github.com/samvera-labs/iiif_print/wiki/Features-List).
+A complete list of features can be found [here](https://github.com/scientist-softserv/iiif_print/wiki/Features-List).
 
 ## Documentation
-A set of helpful documents to help you learn more and deploy IiifPrint can be found on the [Project Wiki](https://github.com/samvera-labs/iiif_print/wiki), including a PCDM model diagram, metadata schema, batch ingest instructions, and more details on installing, developing, and testing the code.
+A set of helpful documents to help you learn more and deploy IiifPrint can be found on the [Project Wiki](https://github.com/scientist-softserv/iiif_print/wiki), including a PCDM model diagram, metadata schema, batch ingest instructions, and more details on installing, developing, and testing the code.
```

Comment:
I'd take out the "including... " because it's only as complete as it is. No need to give false hopes! :D